### PR TITLE
Password To Keychain and reconfig CircleCI

### PR DESCRIPTION
**Pull request by johnedma**
### Motivation :abacus:

Improving overall user experience for DesktopWallet by implementing keytar to save passwords to keychain by opt-in.

### In this PR

- Ability to save password to keychain when creating account, importing account or changing passphrase.
- Adds modal to all password related forms with autofill ability if opted to save password.
- Fix for change password implemented.
- Update to CI config adding in system based dependencies for Linux env to successfully run build and test

### Future Work

- Additional testing

### Caveats 
- Keytar functionality currently only being supported for either newly created accounts or imported accounts. Still needs a check to be added on UnlockWallet to offer the chance for prior users to save their password to keychain then confirm or opt out.

### Screen Shots

| Screen Name                                             | After | Before |
| :------------------------------------------------------ | :----: | :---: |
| UnlockWallet | Screen Shot 2021-06-01 at 11 42 24 AM|Screen Shot 2021-05-18 at 8 28 41 PM|
| CreateAccount |Screen Shot 2021-05-18 at 8 02 31 PM|Screen Shot 2021-05-18 at 8 28 49 PM|

### Testing

Test Suites: 46 passed, 46 total
Tests:       97 passed, 97 total
Snapshots:   6 passed, 6 total
Time:        23.371 s, estimated 24 s
Ran all test suites.
✨  Done in 28.62s.

### CODE REVIEW

File: app/pages/Auth/CreateAccount.view/CreateAccount.view.tsx [hunk 1]
```diff
@@ -88,6 +98,11 @@ const CreateAccountView: FC<CreateAccountViewProps> = ({
             label={t('nameLabel')}
             name="accountName"
           />
+          {values.checkedSavePassword && !values.accountName && (
+            <FormHelperText focused>
+              Account Name is optional, but required to save passphrase
```
Comment (fktt22) [inline app/pages/Auth/CreateAccount.view/CreateAccount.view.tsx:103]: This text should be in i18n

File: app/pages/Auth/CreateAccount.view/CreateAccount.view.tsx [hunk 2]
```diff
@@ -60,14 +64,20 @@ const CreateAccountView: FC<CreateAccountViewProps> = ({
...
         submit: null,
       }}
       onSubmit={handleOnSubmit}
       validationSchema={Yup.object().shape({
-        accountName: Yup.string().max(64, t('accountNameValidation')),
+        accountName: Yup.string()
+          .max(64, t('accountNameValidation'))
+          .when('checkedSavePassword', {
+            is: true,
+            then: Yup.string().required('Account Name is required to save passphrase'),
```
Comment (fktt22) [inline app/pages/Auth/CreateAccount.view/CreateAccount.view.tsx:79]: Use i18n

File: app/pages/Auth/CreateAccount.view/CreateAccount.view.tsx [hunk 3]
```diff
@@ -107,6 +122,26 @@ const CreateAccountView: FC<CreateAccountViewProps> = ({
             type="password"
           />
           <Box pt={1} display="flex">
+            <Box display="flex" alignItems="center" flexDirection="row-reverse">
+              <Box>
+                <Typography display="inline">Save passphrase to keychain?</Typography>
```
Comment (fktt22) [inline app/pages/Auth/CreateAccount.view/CreateAccount.view.tsx:127]: i18n

**Review by fktt22 — DISMISSED** [review]
Fix i18n strings

**Review by BrianCorbin — APPROVED** [review]

**Comment by BrianCorbin** [comment]
LGTM

**Pull request merged**